File: pset6/cash.py

while True:
    try:
        change = float(input("Change owed: "))
        if 0.01 <= change:
            break
    except ValueError:
        continue

coins = round(change * 100)
twenty_five_cent = coins // 25
ten_cent = (coins % 25) // 10
five_cent = ((coins % 25) % 10) // 5
pennies = (((coins % 25) % 10) % 5) / 1
print(int(twenty_five_cent + ten_cent + five_cent + pennies))

File: pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int output = 0;
    float change = get_float("Change: ");
    while (change < 0)
    {
        change = get_float("Change owed: ");
    }
    int coins = round(change * 100);
    while (coins >= 25)
    {
        output++;
        coins = coins - 25;
    }
    while (coins >= 10)
    {
        output++;
        coins = coins - 10;
    }
    while (coins >= 5)
    {
        output++;
        coins = coins - 5;
    }
    while (coins > 0)
    {
        output++;
        coins--;
    }
    printf("%i\n", output);
}